core/sanitizer/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+if(asan)
```

Comment:
Can we move if(asan)..endif() one step above? in core/CMakeLists.txt?

Reply:
Sure.